sys/bloom/bloom.c

```diff
@@ -35,7 +35,7 @@ bloom_t *bloom_new(size_t size, size_t num_hashes, ...)
     }
 
     /* Allocate Bloom array */
-    if (!(bloom->a = calloc(ROUND(size), sizeof(char)))) {
+    if (!(bloom->a = calloc(1, ROUND(size)))) {
```

Comment:
You're right. At lease if we find an esoteric platform were CHAR_BITS < 8 this would matter. Will change.